### PR TITLE
UIU-1686: Accessibility: Form elements must have labels

### PR DESCRIPTION
**Pull request by OleksandrHladchenko1**
Purpose
---------
* Add label for form elements

Approach
-----------
* Added aria-label for form elements

Refs
-----
https://issues.folio.org/browse/UIU-1686

### CODE REVIEW

**Review by mariia-aloshyna — APPROVED** [review]

**Comment by mariia-aloshyna** [comment]
Hi @zburke Could you take a look, please? Thank you!

**Review by zburke — COMMENTED** [review]
Hmmmmmm. It feels like a simpler, and maybe better refactor than adding `aria-label`, would be moving from 
```
<Row>
  <Col xs={12}>
    <b><FormattedMessage id="ui-users.charge.owner.label" /></b>
  </Col>
</Row>
<Row>
  <Col xs={12}>
    <Field
      name="ownerId"
      ...
    />
  </Col>
</Row>
```
to 
```
<Row>
  <Col xs={12}>
    <Field
      name="ownerId"
      label={<FormattedMessage id="ui-users.charge.owner.label" />}
      ...
    />
</Row>
```
What do you think?

**Comment by mariia-aloshyna** [comment]
> Hmmmmmm. It feels like a simpler, and maybe better refactor than adding `aria-label`, would be moving from
> 
> ```
> <Row>
>   <Col xs={12}>
>     <b><FormattedMessage id="ui-users.charge.owner.label" /></b>
>   </Col>
> </Row>
> <Row>
>   <Col xs={12}>
>     <Field
>       name="ownerId"
>       ...
>     />
>   </Col>
> </Row>
> ```
> 
> to
> 
> ```
> <Row>
>   <Col xs={12}>
>     <Field
>       name="ownerId"
>       label={<FormattedMessage id="ui-users.charge.owner.label" />}
>       ...
>     />
> </Row>
> ```
> 
> What do you think?

Hi @zburke ! Looks good, but when I tried to implement this it didn't work. Weird...
Then I figured out that implementation of `Select` component doesn't allow this because of the `aria-label` property, since this applies the `sr-only` class to the label, and the user literally does not see the label. I would propose to make changes in the `Select` component but not in the scope of this task.
https://github.com/folio-org/stripes-components/blob/master/lib/Select/Select.js#L180

**Review by zburke — CHANGES_REQUESTED** [review]
All the form components from stripes-components take a `label` prop which will automatically configure the right styles and a11y attributes; thus `<SomeInput label=... />` should be the first choice when building a form. If that can't be done, the next choice for labeling a field is with a [`<Label>`](https://github.com/folio-org/stripes-components/tree/1ec88ef173b9b69214550287df706e19bb731e6a/lib/Label) that includes a machine-readable `id` prop which can be passed to the `<Field>` it corresponds to with `aria-labelledby`. The last resort, when a field has no visual label, is a human-readable `aria-label` property. 

If converting to `<Field label... />` is too much work in the scope of this ticket, I respect that. But if the approach you take is `aria-label`, the value for that prop should be a human-readable value.

File: src/settings/CommentRequiredForm.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ const Setting = ({
       <Col sm={1}>
         <Field
           name={name}
+          aria-label={name}
```
[the remark this answers is not in the document]
Comment (mariia-aloshyna) [reply src/settings/CommentRequiredForm.js]: Hi @zburke Updated the PR. Please, take a look

**Review by zburke — APPROVED** [review]

**Pull request merged**